specs/fjord/derivation.md

```diff
@@ -48,3 +49,26 @@ be in the same epoch as the the last pre-Fjord blocks, even if these blocks woul
 have these L1-origin timestamps according to pre-Fjord rules. So the same L1 timestamp would be
 shared within a pre- and post-Fjord mixed epoch. This is considered a feature and is not considered
 a security issue.
+
+## Channel Format
```

Comment:
```suggestion
# Brotli Channel Compression
```
or it's part of section "Protocol Parameter Changes". Maybe also a more catchy title :D